frontend/src/HOC/withTest.tsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
```

Comment:
Remove please HOC folder and reuse common folder with an additional structure instead!